client/modules/workspace/src/AppsWizard/AppsFormSchema.ts

```diff
@@ -472,7 +472,7 @@ const FormSchema = (
     queue_name: definition.jobAttributes.execSystemLogicalQueue,
   }) as TTapisSystemQueue;
 
-  if (definition.jobType === 'BATCH') {
+  if (definition.jobType === 'BATCH' && !definition.notes.hideQueue) {
```

Comment:
just checking - does the job submit with defaults set even if hidden here?

Reply:
You were right, and I addressed this in my latest commit.